recipe/meta.yaml

```
@@ -1,37 +1,41 @@
-{% set version = "0.16.10" %}
-
+{% set version = "0.23.8" %}
 
 package:
   name: awscrt
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/a/awscrt/awscrt-{{ version }}.tar.gz
-  sha256: 3101499b479b82f0403f615bf520c1f8b9506818d275e3e28ade25cc6bcdb92b
+  url: https://pypi.org/packages/source/a/awscrt/awscrt-{{ version }}.tar.gz
+  sha256: cba55f3ee80ea3192a0a24e84caad778570250800a59d29ef9efbcd4d1612f2f
   patches:
-    - 0001-Don-t-force-static-linkage.patch
+    - patches/0001-Don-t-force-static-linkage.patch
 
 build:
-  number: 2
+  number: 0
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - python
```

```suggestion
    - python                                 # [build_platform != target_platform]
    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
```
I would leave both here. They don't hurt (since we don't cross-compile) and it also keeps our recipe close to CF. Also, if you are wondering why CF are adding python in `build` and not in `host` (but just when `build_platform != target_platform`), that's because they cross compile). See their docs at https://conda-forge.org/docs/maintainer/knowledge_base/#cross-compilation-examples.

So all in all, when you see stuff like this, you can leave this there untouched and ignore that since `build_platform != target_platform` will never be true for us, and as such will be ignored by conda-build.

---

Added! Thanks for this